packages/pn-personagiuridica-webapp/src/components/IntegrazioneApi/PublicKeys.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Box, Button, Typography } from '@mui/material';
+import { EmptyState, KnownSentiment, useHasPermissions, useIsMobile } from '@pagopa-pn/pn-commons';
+
+import { PNRole } from '../../redux/auth/types';
+import { useAppSelector } from '../../redux/hooks';
+import { RootState } from '../../redux/store';
+
+const PublicKeys: React.FC = () => {
+  const { t } = useTranslation(['integrazioneApi']);
+  const isMobile = useIsMobile();
+  const currentUser = useAppSelector((state: RootState) => state.userState.user);
+  const role = currentUser.organization?.roles ? currentUser.organization?.roles[0] : null;
+  const userHasAdminPermissions = useHasPermissions(role ? [role.role] : [], [PNRole.ADMIN]);
+
+  const isAdminWithoutGroups = userHasAdminPermissions && !currentUser.hasGroup;
+
+  return (
+    <Box mt={5}>
```

Comment:
This is needed? We can move mt on the stack below and remove this

Reply:
I need the box because the stack below has direction row